src/sqlancer/stonedb/gen/StoneDBTableCreateGenerator.java

```diff
@@ -145,15 +149,14 @@ private void appendTableOptions() {
     }
 
     private void addCommonErrors(ExpectedErrors list) {
-        list.add("The storage engine for the table doesn't support");
+        // java.sql.SQLSyntaxErrorException: BLOB/TEXT column 'c0' used in key specification without a key length
```

Comment:
i think it's okay to remove those comments , but it's your call to keep them or remove

Reply:
How about leaving it here to know each expected error's source?